src/utils/env.ts

```diff
@@ -1,15 +1,19 @@
-import { Client } from '@discordjs/core';
+import { Client, GatewayIntentBits } from '@discordjs/core';
 import { REST } from '@discordjs/rest';
 import { WebSocketManager } from '@discordjs/ws';
 import { env } from 'node:process';
 
 export const botToken = env.BOT_TOKEN ?? '';
 export const guildId = env.GUILD_ID ?? '';
-
+export const apiKey = env.IMAGE_API ?? '';
+export const apiUrl = env.IMAGE_URL ?? '';
 export const rest = new REST({ version: '10' }).setToken(botToken);
 export const gateway = new WebSocketManager({
     token: botToken,
-    intents: 0,
+    intents:
+        GatewayIntentBits.Guilds |
```

Comment:
```suggestion
```
I believe this intent can be safely removed as it isn't used in this case.

Reply:
Done, left in while testing